miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/util/BulkPropagateTableBackend.java

```diff
@@ -1,20 +1,23 @@
 package uk.ac.bbsrc.tgac.miso.webapp.util;
 
 import java.io.IOException;
+import java.util.ArrayList;
 import java.util.List;
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
 import org.springframework.ui.ModelMap;
 import org.springframework.web.servlet.ModelAndView;
 
+import uk.ac.bbsrc.tgac.miso.core.data.Identifiable;
+
 /**
  * Create a Handsontable for propagating a particular entity type
  *
  * @param <ParentModel> The database model for the entity from which the model can be propagated
  * @param <Dto> The DTO for the entity being propagated
  */
-public abstract class BulkPropagateTableBackend<ParentModel, Dto> extends BulkTableBackend<Dto> {
+public abstract class BulkPropagateTableBackend<ParentModel extends Identifiable, Dto> extends BulkTableBackend<Dto> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00119)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00119)